build.gradle

```diff
@@ -101,12 +101,14 @@ grazel {
             androidTest {
                 robolectric {
                     version = "4.4"
+                    stripPrefix = "robolectric-bazel-4.4"
```

Comment:
Instead of adding manually adding properties like this can we reuse the `httpRepository` or `gitRepository` structure created for Kotlin and Bazel Common rules?

For example, [this](https://github.com/grab/Grazel/blob/5677c1b2d29b554b0577bdf0780dc1a74e1db1ce/grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/bazel/rules/RepositoryRules.kt#L88) and usage like [this](https://github.com/grab/Grazel/blob/5677c1b2d29b554b0577bdf0780dc1a74e1db1ce/grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/extension/KotlinExtension.kt#L141-L169).